httphandler/roundtripper_decorators.go

```diff
@@ -122,6 +129,33 @@ func (os optionsHandler) RoundTrip(req *http.Request) (resp *http.Response, err
 	return
 }
 
+type statusHandler struct {
+	roundTripper http.RoundTripper
+}
+
+func (sh statusHandler) RoundTrip(req *http.Request) (resp *http.Response, err error) {
+
+	if strings.Contains(req.Method, http.MethodGet) && strings.Contains(strings.ToLower(req.URL.Path), HealthCheckEndpoint) {
+		resp, err = sh.roundTripper.RoundTrip(req)
```

Comment:
We should discuss if HC should be passed further or should it be terminated in akubra